packages/embeds/embed-core/src/__tests__/embed-iframe-methods.test.ts

```diff
@@ -0,0 +1,144 @@
+import { describe, it, expect, vi, beforeEach, afterEach } from "vitest";
+
+import { fakeCurrentDocumentUrl, nextTick } from "../embed-iframe/__tests__/test-utils";
+
```

Comment:
Created this from embed-iframe.test.ts

Motivation behind this is to separate the tests that mock embedStore to a separate file as they interfare with other tests in embed-iframe.test.ts which don't use mock. This is anyway a good approach to have shorter test files, as they are easy to maintain